### ISSUE TITLE
ICD-10 and/or CPT code inclusion

### ISSUE THREAD

**Issue by SQLDataNinja** [synthetichealth/synthea#403]
Hello,
Is there a chancee that inclusion of ICD-10 and/or CPT Codes to patient encounter and procedure data might be on the roadmap for this awesome project?

Thanks!

**Comment by jawalonoski** [comment]
Sort answer is "only if it is included by the user as part of a value set."

See #347 and #380.

In general, the idea is that inside a [module](https://github.com/synthetichealth/synthea/wiki/Generic-Module-Framework) a user may specify a FHIR `ValueSet` instead of a single code and the system will randomly use different codes from that value set in the records. That value set can be provided locally or from a service like VSAC (https://vsac.nlm.nih.gov).

We do not plan to embed ICD-10 or CPT in the repository due to licensing issues (both real and/or perceived).

**Comment by jawalonoski** [comment]
Closed with #709.   

In order for ValueSets to be used, you need to specify a FHIR terminology server using the `generate.terminology_service_url` property and have a module with ValueSet specified.

ValueSets can be specified in any [State](https://github.com/synthetichealth/synthea/wiki/Generic-Module-Framework%3A-States) that contains a `code`. Just add a `value_set` attribute in addition to the system, code, and display.

For example:
```json
{
  "system": "SNOMED-CT",
  "code": "2501000087107",
  "display": "MRI of right knee with contrast",
  "value_set": "http://snomed.info/sct?fhir_vs=ecl/<<2491000087104"
}
```

**Comment by jerrydeng** [comment]
The example value_set: "http://snomed.info/sct?fhir_vs=ecl/<<2491000087104" links to a 404 page.  I was wondering if you can give an example of local value_set or what would a terminology server/service link look like.  Does the server need to be specific query URL and give a simple txt/json/xml response?  What would be the lowest hanging fruit approach?  Modify all modules to include some ICD directly on value_set attribute for each encounter codes?
